packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -45,6 +54,8 @@ export interface FunctionExecution extends Terminable {
   debugSession?: vscode.DebugSession;
 }
 
+export const FUNCTION_TYPE_ERROR = 'Unable to determine type of exectucting function.';
+
```

Comment:
FunctionService looks to be a singleton, but it's missing a private constructor.  Since you are making changes to this file, could you add a private constructor?